tests/RobotFramework/tests/cumulocity/log/log_operation_child.robot

```diff
@@ -0,0 +1,60 @@
+*** Settings ***
+Resource            ../../../resources/common.resource
+Library             Cumulocity
+Library             DateTime
+Library             ThinEdgeIO
+
+Suite Setup         Custom Setup
+Test Teardown       Get Logs
+
+Test Tags           theme:c8y    theme:log
+
+
+*** Test Cases ***
+Successful log operation
+    ${end_timestamp}=    get current date    result_format=%Y-%m-%dT%H:%M:%S+0000
+    ${operation}=    Cumulocity.Create Operation
+    ...    description=Log file request
+    ...    fragments={"c8y_LogfileRequest":{"dateFrom":"1970-01-01T00:00:00+0000","dateTo":"${end_timestamp}","logFile":"example","searchText":"first","maximumLines":10}}
+    ${operation}=    Operation Should Be SUCCESSFUL    ${operation}    timeout=120
+
+
+*** Keywords ***
+Setup Child Device
+    ThinEdgeIO.Set Device Context    ${CHILD_SN}
+    Execute Command    sudo dpkg -i packages/tedge_*.deb
+    Execute Command    sudo dpkg -i packages/tedge-log-plugin*.deb
+
+    Execute Command    sudo tedge config set mqtt.client.host ${PARENT_IP}
+    Execute Command    sudo tedge config set mqtt.client.port 1883
+
+    ThinEdgeIO.Transfer To Device    ${CURDIR}/tedge-log-plugin.toml    /etc/tedge/plugins/tedge-log-plugin.toml
+    ThinEdgeIO.Transfer To Device    ${CURDIR}/example.log    /var/log/example/
+    Execute Command    chown root:root /etc/tedge/plugins/tedge-log-plugin.toml /var/log/example/example.log
+    ThinEdgeIO.Stop Service    tedge-log-plugin
+    Execute Command    cmd=sed -i 's|ExecStart=.*|ExecStart=/usr/bin/tedge-log-plugin --root te --device "device/${CHILD_SN}//"|g' /lib/systemd/system/tedge-log-plugin.service && sudo systemctl daemon-reload
```

Comment:
We have a plan to have `root.topic` and `device.topic` in tedge config so that `tedge-log-plugin` can consume them in addition to giving them as arguments like here. Do you think this improvement can be done after we release `tedge-log-plugin`?

Reply:
I don't think we need to wait for the root and device to be configurable via configuration. I'm happy to make the change to the test once the feature lands. We don't have to necessarily promote usage of the tedge-log-plugin for child devices immediately, but its good to know that it works first by just starting it via command line flags.

Reply:
FYI: https://github.com/thin-edge/thin-edge.io/pull/2231 is renaming the arguments:

- `--device` => `--device-topic-id`
- `--root` => `--mqtt-topic-root`

Reply:
@rina updated the flags to match.